app/models/spotlight/resources/iiif_harvester.rb

```diff
@@ -1,45 +1,43 @@
-module Spotlight::Resources
-  # harvest Images from IIIF Manifest and turn them into a Spotlight::Resource
-  # Note: IIIF API : http://iiif.io/api/presentation/2.0
-  class IiifHarvester < Spotlight::Resource
-    self.weight = -5000
-
-    after_save :harvest_resources
+require 'iiif/presentation'
+
+module Spotlight
+  module Resources
+    # harvest Images from IIIF Manifest and turn them into a Spotlight::Resource
+    # Note: IIIF API : http://iiif.io/api/presentation/2.0
+    class IiifHarvester < Spotlight::Resource
+      self.weight = -5000
+
+      validate :valid_url?
+
+      def to_solr
+        return to_enum(:to_solr) { 0 } unless block_given?
```

Comment:
The `0` causes the `total` number in the reindex monitor panel to be 0.

This creates a pretty odd display where you get text like `6 of 0 total items`.

This is all in a background job so maybe we don't care and we can just ignore the fact that getting the count will cause the traversal of the entire hierarchy.

Any thoughts?